### PR TITLE
Prototype extensions disabled compat

### PR DESCRIPTION
**Pull request by jbrown**

### CODE REVIEW

**Comment by ef4** [comment]
Thanks. 

I think you have a typo. `observes` -> `observer`

**Comment by jbrown** [comment]
Sorry, long day and editing in github is never a good combo. :flushed:

Should be fixed now.

File: app/components/liquid-child.js [hunk 1]
```diff
@@ -2,14 +2,14 @@ import Ember from "ember";
 export default Ember.Component.extend({
   classNames: ['liquid-child'],
 
-  updateElementVisibility: function() {
+  updateElementVisibility: Ember.on('willInsertElement', Ember.observer('visible', function() {
```
Comment (sukima) [inline app/components/liquid-child.js]: I didn't know you could nest 'em like that. :astonished:

Comment (jbrown) [inline app/components/liquid-child.js]: @sukima They both return the function you pass them.

**Comment by jbrown** [comment]
Looks like this is irrelevant on master. Closing...

**Pull request closed without merging**